services/QuillLMS/app/models/user.rb

```diff
@@ -473,7 +473,7 @@ def self.sorting_name_sql
   end
 
   def capitalize_name
-    self.name = ::CapitalizeNames.capitalize(name)
+    self.name = ::CapitalizeNames.capitalize(name, skip_van_space: true, skip_dit_space: true)
```

Comment:
I think these settings will give undesired behavior
```
1] pry(main)> CapitalizeNames.capitalize("Alexander Van Humboldt")
=> "Alexander van Humboldt"
[2] pry(main)> CapitalizeNames.capitalize("Alexander Van Humboldt", skip_van_space: true)
=> "Alexander Van Humboldt"
```
Instead, keep the original setting
`self.name = ::CapitalizeNames.capitalize(name)`

and then do some post-processing that checks for the first word in the name for "mac, dit, etc...." and capitalizes them.